src/minMaxLength.js

```diff
@@ -44,10 +44,27 @@ const minMaxLength = (options) => {
 		return maxVal === null ? true : el.value.length <= parseInt(maxVal);
 	}
 
-    function postprocessMessage(msg) {
+    function getMinMaxValues(field) {
+        let minVal = settings.minToken;
+        let maxVal = settings.maxToken;
+        if (field.inputEls) {
+            const invalidEls = field.inputEls.filter(el => !meetsMin(el) || !meetsMax(el));
+            if (invalidEls.length) {
+                minVal = invalidEls[0].getAttribute(settings.minAttr);
+                maxVal = invalidEls[0].getAttribute(settings.maxAttr);
+            }
+        }
+        return {
+            minVal,
+            maxVal
+        }
+    }
+
+    function postprocessMessage(msg, field = {}) {
```

Comment:
Unless you think it adds clarity, you could ditch the `else` and just put those `msg` replaces after the `if`